source/secure_tunneling.c

```diff
@@ -623,9 +664,11 @@ static void s_aws_secure_tunnel_on_stream_reset_received(
         return;
     }
 
-    int result = AWS_OP_SUCCESS;
     if (s_aws_secure_tunnel_stream_id_match_check(secure_tunnel, message_view->service_id, message_view->stream_id)) {
-        result = s_aws_secure_tunnel_set_stream(secure_tunnel, message_view->service_id, INVALID_STREAM_ID, 0);
+        int result = AWS_OP_SUCCESS;
```

Comment:
Same comment here.  We're conflating AWS_OP_SUCCESS/AWS_OP_ERR with error codes.  Same change applies